pyproject.toml

```diff
@@ -49,15 +49,16 @@ artifacts = [
   "*.pyccel",
 ]
 include = [
-  "include",
   "*.py",
   "*.pyh",
   "*.pyccel",
   "pyccel/parser/grammar/*.tx",
   "pyccel/stdlib/*/*.h",
   "pyccel/stdlib/*/*.c",
   "pyccel/stdlib/*/*.f90",
-  "pyccel/extensions/STC/include/*/*.c"
+  "pyccel/extensions/STC/include",
+  "pyccel/extensions/STC/include/*/*.c",
```

Comment:
Are both of these lines needed?

Reply:
I think the first line will add all folders and files, so I will remove the second line.

Reply:
Fixed : db739e5